install/installer/pkg/config/v1/config.go

```diff
@@ -113,6 +113,8 @@ type Config struct {
 
 	Customization *[]Customization `json:"customization,omitempty"`
 
+	Components *Components `json:"components,omitempty"`
```

Comment:
I don't have a strong opinion here, but: did you think about moving this under an `advanced` section?
The idea is to mirror the KOTS config, including the "reduce service" that we offer for changes that go into this section: "only touch when advised or you know what you're doing".

Reply:
I didn't, although I think there's probably quite a few things which are in the config already that are "advanced" topics that we've not done so with - customization and the resource limits jump to mind.

My personal feeling is that we should keep the config as objective to the section, rather than differentiating between basic, advanced and experimental sections.

But, like you, I don't have a strong opinion here either. I'll let our friend @corneliusludmann have the final say